packages/core/src/components/Tabs/Tabs.jsx

```diff
@@ -190,37 +179,31 @@ export class Tabs extends React.PureComponent {
    * handleTabClick, so passing an interpolated hashbang plus selected Tab
    * string instead. 
    */
-  tabSwitchLeft(evt, panelId) {
+  tabSwitcher(pressed, evt, panelId) {
```

Comment:
Seems like this could just become `handleTabKeyDown`, no?

Reply:
I'm opting to rename it to handleTabKeyFocus, where handleTabKeyDown is already defined, but feel it does a good job explaining what the method does. Will push that change after testing, and re-run the build.

Reply:
@1Copenut Sorry, to clarify: I was suggesting that what is currently in `handleTabKeyDown` seems redundant and `tabSwitcher` could essentially replace it, with a few minor changes (ie. moving the `evt.key` logic into this method). 

Let me know if it'd be easier to chat about this IRL.

Reply:
@sawyerh Yes, after reading your comment, we could just make that one method easily. I'll follow up with you about a time to chat IRL.